index.html

```diff
@@ -2428,58 +2428,71 @@
         </h2>
-        <p>
-          The <a>user agent</a> MUST NOT share information about the user to
-          the web page (such as the shipping address) without user consent.
+        <p class="ednote">
+          This section is a placeholder to record privacy considerations as we
+          gather them through working group discussion.
         </p>
+        <section>
+          <h2>
+            Exposing user information
+          </h2>
+          <p>
+            The <a>user agent</a> MUST NOT share information about the user to
+            the web page (such as the shipping address) without user consent.
+          </p>
+        </section>
+        <section>
+          <h2>
+            Exposing available payment methods
+          </h2>
+          <p>
+            A page might try to call the payment request API repeatedly with only
+            one payment method identifier to try to determine what payment
+            methods a <a>user agent</a> has installed. There may be legitimate
+            scenarios for calling repeatedly (for example, to control the order
+            of payment method selection). The fact that a successful match to a
+            payment method causes a user interface to be displayed mitigates the
+            disclosure risk. Implementations may also require a user action to
+            initiate a payment request or they may choose to rate limit the calls
+            to the API to prevent too many repeated calls.
+          </p>
+        </section>
       </section>
-      <section>
-        <h2>
-          Exposing available payment methods
-        </h2>
-        <p>
-          A page might try to call the payment request API repeatedly with only
-          one payment method identifier to try to determine what payment
-          methods a <a>user agent</a> has installed. There may be legitimate
-          scenarios for calling repeatedly (for example, to control the order
-          of payment method selection). The fact that a successful match to a
-          payment method causes a user interface to be displayed mitigates the
-          disclosure risk. Implementations may also require a user action to
-          initiate a payment request or they may choose to rate limit the calls
-          to the API to prevent too many repeated calls.
-        </p>
+      <section class="informative">
+          <h2>Accessibility Considerations</h2>
+          <p>
+          This specification has no defined user interface. In addition, there are no specific accessibility requirements on implementations. However, to the extent that an
+          implementation provides user interactions in support of this specification, the implementation must ensure that the interface for those interactions is exposed to the
+          platform accessibility API. Moreover, implementors should take into consideration the needs of their users with varying abilities when designing solutions that
```

Comment:
This is not consistent with itself. There is a concrete accessibility requirement named: That anything exposed in the UI is exposed to the accessibility API.

This seems necessary but not sufficient. Information needs to meet accessibility requirements for users other than those who rely on the accessibility API, too.